app/Controllers/entryController.php

```diff
@@ -78,10 +78,10 @@ public function readAction() {
 				$get = substr($get, 2);
```

Comment:
```suggestion
				$get = (int)substr($get, 2);
```

And remove the casts further down